Sources/Extensions/UIView+Constraints.swift

```diff
@@ -8,14 +8,50 @@
 import UIKit
 
 extension UIView {
+    /// Constrain this view to the it's parent view.
+    ///
+    /// - parameter insets: The insets that give a some padding if needed.
+    public func constraintToSuperview(insets: UIEdgeInsets = .zero) {
```

Comment:
Kleine opmerking qua naamgeving: in deze context gebruik is het volgens mij `constrainToSuperView()`. 
Het werkwoord is `to constrain`, het zelfstandig naamwoord `a constraint`.
(idem voor de methods hieronder)

Reply:
Goei punt!